gcp/table_gcp_apikeys_key.go

```diff
@@ -0,0 +1,208 @@
+package gcp
+
+import (
+	"context"
+
+	"github.com/turbot/go-kit/types"
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+	"google.golang.org/api/apikeys/v2"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+)
+
+//// TABLE DEFINITION
+
+func tableGcpApikeysKey(_ context.Context) *plugin.Table {
```

Comment:
```suggestion
func tableGcpApiKeysKey(_ context.Context) *plugin.Table {
```

We should be sure to follow camel case wherever the service name appears